src/Aside/Aside.js

```diff
@@ -3,63 +3,21 @@ import Controls from '../Controls/Controls.js';
 import AsideBody from '../AsideBody/AsideBody.js';
 import './Aside.css';
 
-class Aside extends Component {
-  constructor(props) {
-    super(props);
-    let storedCardNames = [];
-    if (props.userCardsData.length > 0) {
-      storedCardNames = props.userCardsData.map(card => card.cardName);
-    }
-    this.state = {
-      cardNames: storedCardNames,
-      hasDuplicates: false,
-      isInvalidCardName: false
-    };
-  }
-
-  addCardToList = (cardName) => {
-    if (!this.state.cardNames.includes(cardName)) {
-      const cardNames = this.state.cardNames.concat([cardName]);
-      this.setState({
-        cardNames: cardNames,
-        hasDuplicates: false,
-        isInvalidCardName: false,
-      });
-      this.props.addUserCard(cardName);
-    } else {
-      this.setState({ hasDuplicates: true });
-    }
-  }
-
-  removeListItem = (indexToRemove) => {
-    const [...newCardNames] = this.state.cardNames;
-    newCardNames.splice(indexToRemove, 1)
-    this.setState({ cardNames: newCardNames })
-    this.props.removeUserCard(indexToRemove);
-  }
-
-  throwInvalidCardNameError = () => {
-    this.setState({ isInvalidCardName: true });
-  }
-  
+class Aside extends Component {  
```

Comment:
We can potentially change Aside to a functional component since it no longer has state or methods.